src/frontend/src/catalog/table_catalog.rs

```diff
@@ -52,8 +53,12 @@ pub struct TableCatalog {
     /// on this to derive an append-only stream plan
     pub appendonly: bool,
 
-    // owner of table
+    /// Owner of the table.
     pub owner: String,
+
+    /// Mapping from vnode to parallel unit. Indicates data distribution and partition of the
+    /// table.
+    pub vnode_mapping: Option<Vec<u32>>,
```

Comment:
```suggestion
    pub vnode_mapping: Option<Vec<ParallelUnitId>>,
```

Perhaps `ParallelUnitId` should be moved to `common`